inputs_demand.md

```diff
@@ -99,9 +101,13 @@ Fuel preference elasticities are specified in [A32.fuelprefElasticity.csv](https
 
 Multipliers used to determine the value of time in transit are specified in [A54.tranSubsector_VOTT_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.tranSubsector_VOTT_revised.csv).
 
+#### Residential floorspace parameters
+
+The parameters for the estimation of residential floorspace demand are listed in [A44.flsp_param.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.flsp_param.csv)
```

Comment:
I went back and checked the Gompertz CMP: this CSV file doesn't exist.  Instead it looks like the params are fit as part of the data system process (and saved in `L144.flsp_param`).  I'm not sure what to say here then.  Maybe just describe the fitting process and note USA and China have more detailed data sets used?

Reply:
Yes, I have explained that the analysis is developed within the model DS, and specified that the values for the US have been estimated outside using subnational data.